bolt04/onion-test-multi-frame.json

```diff
@@ -0,0 +1,42 @@
+{
+  "comment": "This is a simple testcase in which we only use v0 payloads, and all hops have single frame payloads",
```

Comment:
nit: this title should be updated if this is supposed to be the multi-frame test vector (as indicated by the file name). Maybe rename that entirely to `onion-test-variable-length.json`?